support/power-platform/power-automate/flow-approval-cds-provisioning-errors.md

```diff
@@ -17,67 +17,67 @@ _Original KB number:_ &nbsp; 4513672
+To resolve this issue, tenant administrators need to go to the **Enterprise Applications** tab under **Azure Active Directory** in [Microsoft Azure](https://ms.portal.azure.com) to ensure application 00000007-0000-0000-c000-000000000000 (Dataverse or Dynamics CRM Online) is enabled for users to sign in, and any relevant Conditional Access policies grant the necessary access to users expecting to use Power Automate Approvals.
 
-:::image type="content" source="media/flow-approval-cds-provisioning-errors/properties-settings.png" alt-text="Screenshot to set the Enabled for users to sign-in option to Yes.":::
+:::image type="content" source="media/flow-approval-cds-provisioning-errors/properties-settings.png" alt-text="Screenshot that shows how to set the Enabled for users to sign-in option to Yes.":::
 
 ## Error "Database is disabled"
 
 > The Dataverse database for this environment is disabled
 
-The Dataverse instance has been disabled in this environment. It isn't expected, and is related to the expiration of all Power Automate & Dataverse plans within your AAD tenancy. To ensure the database can be enabled, make sure at least one user has active plans.
+The Dataverse instance has been disabled in this environment. It isn't expected and is related to the expiration of all Power Automate & Dataverse plans within your Azure Active Directory (AD) tenancy. To ensure the database can be enabled, make sure at least one user has active plans.
 
 ## Error "Database isn't ready yet"
 
 > The Dataverse Database for this environment is not ready yet.
 
-The database for this instance is still being provisioned, or has failed provisioning. Rerunning a flow that uses approvals will attempt to reprovision the instance.  
+The database for this instance is still being provisioned or has failed provisioning. Rerunning a flow that uses approvals will attempt to reprovision the instance.
 
 ## Error "User has no permission to create database"
 
 > The current user doesn't have permissions to create a Dataverse database for this environment.
 
-For non-Default Power Automate and Power Apps environments, only environment admins can directly (through the Power Apps Admin portal) or indirectly (through Power Automate Approvals) create the Common Data Service database.
+For non-default Power Automate and Power Apps environments, only environment admins can directly (through the Power Apps Admin portal) or indirectly (through Power Automate Approvals) create the Dataverse database.
 
 Either an administrator must:
 
-- Create the environment manually from the Flow Admin portal.
-- Create and run an Approvals Flow.
+- Create the environment manually from the Power Apps Admin portal.
```

Comment:
I'd rephrase this to Power Platform Admin portal since its common

Reply:
Hi @hamenon-ms, I updated the sentence as per your suggestion. Thank you for reviewing the details!